graphql/init.lua

```diff
@@ -1 +1 @@
-return {VERSION = 'unknown'}
+return {_VERSION = require('graphql.version')}
```

Comment:
Let's use the following approach: if module already has a version handle, we will not remove it. (It would be the best to add an issue about `.VERSION` deprecation, but let's do it later.) So let's preserve two variables for a time (but let their values be equal -- let the old version use the new approach).

Reply:
Done